client/src/components/forms/form-wrapper.component.test.tsx

```diff
@@ -1,20 +1,25 @@
 import { describe, expect, it } from 'vitest';
 import { FormWrapper } from './form-wrapper.component';
 import { render, screen } from '~/test/utils';
-import { LoginForm } from '~/accounts/authentication/login-form.component';
-
-let loginUser: () => void;
 
 describe('form-wrapper', () => {
-  beforeEach(() => {
-    loginUser = vi.fn();
-  });
-  it('should render form-wrapper on the page with login form', () => {
+  it('should render form-wrapper on the page with a form', () => {
     render(
       <FormWrapper>
-        <LoginForm loginUser={loginUser} />
+        <form aria-label="Add your details">
```

Comment:
This is much better, you don't need the `aria-label` though as you don't actually use it, but it isn't causing any harm.